src/app/retrospective/sprint-list/sprint-list.component.ts

```diff
@@ -22,14 +22,27 @@ export class SprintListComponent implements OnInit {
     dataSource: SprintListDataSource;
     sprintStates = SPRINT_STATES_LABEL;
     dateFormat = DATE_FORMAT;
+    lastScrollTop: number;
 
-    constructor(
-        private retrospectiveService: RetrospectiveService,
-        private snackBar: MatSnackBar,
-        private router: Router,
-        private utils: UtilsService,
-        private changeDetectorRefs: ChangeDetectorRef
-    ) {
+    @HostListener('window:scroll', [])
+    onWindowScroll() {
+        const currentHeight = window.innerHeight + window.scrollY;
+
+        // If the user has scrolled to 70% of the page and is scrolling down, add more data
+        if (currentHeight > (document.body.offsetHeight * 0.7) && window.pageYOffset > this.lastScrollTop) {
```

Comment:
Make `0.7` as a constant and also add some reference links (if any) for all the calculations so as to make the code more readable